demo/README.md

```diff
@@ -8,7 +8,7 @@ then build the circuit and setup the proving/verifying key, and then generate a
 finally verify the proof with the verifying key.
 
 ```bash
-bin/zkmove run -s examples/scripts/add.move
+zkmove run -s examples/scripts/add.move
```

Comment:
relative path(bin/zkmove) is better here?

Reply:
bin/ and examples/ is under same parent dir demo/.  Using bin/zkmove can save the interpretation for the execution environment :)